libs/platform/src/lib/approval-flow/approval-flow.component.ts

```diff
@@ -300,6 +308,10 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
             });
     }
 
+    getMetadata(nodeId: string): ApprovalGraphNodeMetadata {
```

Comment:
let's call it more obvious, e.x `getNodeMetadaById`,
also please add property description comments

Reply:
It will be a breaking change, just fyi.

Reply:
why breaking change? this is a new method

Reply:
my miss, sorry, you're right.

Reply:
Done